Resources/public/js/views/fields/ez-date-editview.js

```diff
@@ -55,6 +55,19 @@ YUI.add('ez-date-editview', function (Y) {
         },
 
         /**
+         * Checks whether the field contains a valid timestamp.
+         *
+         * @method _containsValidTimestamp
+         * @protected
+         * @return {Boolean}
+         */
+        _containsValidTimestamp: function () {
```

Comment:
Nitpick: You should check how it is done in Time and DateAndTime with the `_isFieldEmpty` method. Maybe it could be nice to have have some coherency between them.